### PR TITLE
GH-22 - header component

### PR DESCRIPTION
**Pull request by chuettenrauch**
- header component with logo and user image
- account menu with logout functionality
- toast containers
- some small changes on overall colors

### CODE REVIEW

File: frontend/src/components/HeaderNav.tsx [hunk 1]
```diff
@@ -1,19 +1,70 @@
...
+    const navigate = useNavigate();
+
+    const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+    const open = Boolean(anchorEl);
+
+    const onLogout = async () => {
+        await UserApi.logout();
+
+        toast.success("Successfully logged out.");
+
```
Comment (bitbytejoy) [inline frontend/src/components/HeaderNav.tsx:31]: If I understand this correctly, might want to do a setUser(null), where setUser comes from the useUserContext()

**Review by bitbytejoy — APPROVED** [review]
lgtm

File: frontend/src/components/HeaderNav.tsx (hunk 1 above)
Comment (chuettenrauch) [reply frontend/src/components/HeaderNav.tsx:31]: Good point 👍  I'll change that

Comment (chuettenrauch) [reply frontend/src/components/HeaderNav.tsx:32]: Done

**Pull request merged**